### DISCUSSION TITLE
Add Checkbox for regex search

### DISCUSSION THREAD

**Discussion by ivan200** [rejetto/hfs#658] [Q&A] [answered]
i have server with many (200k+) files
files have names like this:
`eastrun[df.cpm]02.43.488(uN-DeaD!Enter.Russia).dm_68`

most people search there like this
`eastrun[df.cpm]`
and no files are found

or
`(uN-DeaD!Enter.Russia)`
and every file from my server are found

Could you please use the search for a simple search (without regex) and add the unchecked "regex" checkbox to search for regular expressions?

**Comment by rejetto** [comment]
ok, i will introduce an option for it.

just for information: "regex" are a similar thing, but these are not regex.

**Comment by rejetto** [comment]
i'm already doing this, but in the meantime, you may want to know how to search for parenthesis with today's version
https://github.com/rejetto/hfs/wiki/Wildcards#escaping

**Comment by rejetto** [comment, marked as answer]
version 0.54.0-alpha1 is out and includes this
